### PR TITLE
Fix logic of sharing wallet info when sdk is not blocked

### PR DESCRIPTION
**Pull request by subhod-i**
Fixed the broken logic from PR https://github.com/aeternity/superhero-wallet/pull/2185

### CODE REVIEW

File: src/background/wallet.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ const addAeppConnection = async (port) => {
   const connection = new BrowserRuntimeConnection({ port });
   const clientId = aeSdk.addRpcClient(connection);
   await aeSdk.shareWalletInfo(clientId);
-  const shareWalletInfo = setInterval(() => aeSdkBlocked && aeSdk.shareWalletInfo(clientId), 3000);
+  const shareWalletInfo = setInterval(
+    () => !isAeSdkBlocked && aeSdk.shareWalletInfo(clientId), 3000,
+  );
```
Comment (pstachel) [inline src/background/wallet.js:22]:
```suggestion
  const shareWalletInfo = setInterval(
    () => !isAeSdkBlocked && aeSdk.shareWalletInfo(clientId),
    3000,
  );
```

**Review by pstachel — APPROVED** [review]

**Review by konradodo — APPROVED** [review]

**Review by konradodo — APPROVED** [review]

**Comment by CharisSiarampalis** [comment]
@subhod-i I am getting more errors on the console compared to dev

image

**Comment by Liubov-crypto** [comment]
@subhod-i for me it looks good. No additional errors compared to dev. 

@CharisSiarampalis I have the same 404 errors on dev branch.

**Comment by subhod-i** [comment]
> @subhod-i for me it looks good. No additional errors compared to dev.
> 
> @CharisSiarampalis I have the same 404 errors on dev branch.

Thanks, @Liubov-crypto for the confirmations. @CharisSiarampalis it's the same error as I see in the screenshot.

**Pull request merged**